webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -167,7 +169,7 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
         </div>
       </div>
       <div class="grader-navigator">
-        <div><a class="user-list-link" href="${this.userListUrl}" title="${this.assignmentsI18n["nav.list"]}">${this.assignmentsI18n["nav.list"]}</a></div>
+        <div><a class="user-list-link" href="${this.userListUrl}" title="${this.assignmentsI18n["nav.list"]}" @click=${this.toStudentList}>${this.assignmentsI18n["nav.list"]}</a></div>
```

Comment:
I'm confused here. Why add a click handler when the href is already set? I know you must have a reason - I'm just asking.

Reply:
The aim is to not navigate when Feedback or Private Notes are changed but are not save. If I remove href, the link style is lost and it looks like normal text. Any suggestion to solve this in a better way would be great.